app/host/attendance/AttendeePresenceTable.tsx

```diff
@@ -111,44 +158,70 @@ export default function AttendeePresenceTable({
 					<tbody>
 						{joined
 							.filter((attendee, _i) => {
-								let show = true;
+								let showStatus = false;
 								if (
-									selectedFilter === "Absent" &&
+									isPresentSelected &&
 									statuses[attendee.attendee] === "Present"
 								) {
-									show = false;
+									showStatus = true;
 								}
 								if (
-									selectedFilter === "Present" &&
+									isAbsentSelected &&
 									statuses[attendee.attendee] !== "Present"
 								) {
-									show = false;
+									showStatus = true;
 								}
-								if (selectedFilter === "Late") {
-									// TODO: ADD "LATE"
-									show = false;
+
+								let showType = false;
+
+								if (
+									isForeignSelected &&
+									!totalAttendeesList
+										.map((x) => x.attendee)
+										.includes(attendee.attendee)
+								) {
+									showType = true;
 								}
+
+								if (
+									isRegisteredSelected &&
+									totalAttendeesList
+										.map((x) => x.attendee)
+										.includes(attendee.attendee)
+								) {
+									console.log("AHH2");
+									showType = true;
+								}
+
 								if (
 									!attendee
 										.profiles!.username.toLowerCase()
 										.includes(searchContent.toLowerCase())
-								)
-									show = false;
+								) {
+									showStatus = false;
+								}
 
-								return show;
+								return showStatus && showType;
 							})
 							.map((attendee, i) => {
 								return (
 									<tr key={attendee.attendee}>
 										<th>{i + 1}</th>
-										<td>{attendee.profiles!.username}</td>
+										<td>
+											{attendee.profiles!.username}
+											{!totalAttendeesList
+												.map((x) => x.attendee)
+												.includes(attendee.attendee) && (
+												<p className="opacity-50">(Unregistered)</p>
```

Comment:
I’m thinking of a badge-like pill to “tag” unregistered people. Kind of like “pending”:
![IMG_3352](https://github.com/EvanZhouDev/zyma/assets/66848002/f5f90080-e7a6-41a3-807f-a108ac71c49e)